helm/cluster-autoscaler-app/templates/service.yaml

```diff
@@ -4,13 +4,20 @@ metadata:
   name: {{ .Values.name }}
   namespace: {{ .Values.namespace }}
   labels:
-    giantswarm.io/service-type: "{{ .Values.serviceType }}"
-    app: {{ .Values.name }}
+    {{- include "labels.common" . | nindent 4 }}
+    # kubernetes.io/cluster-service are kept to be displayed in kubectl cluster-info
     kubernetes.io/cluster-service: "true"
     kubernetes.io/name: {{ .Values.name }}
+    giantswarm.io/monitoring: "true"
+  annotations:
+    giantswarm.io/monitoring-path: /metrics
+    giantswarm.io/monitoring-port: "8085"
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "8085"
+    prometheus.io/path: "/metrics"
 spec:
   selector:
-    app: {{ .Values.name }}
+    {{- include "labels.selector" . | nindent 4 }}
   ports:
   - name: "metrics"
     port: 8085
```

Comment:
Would it make sense to make the port configurable, and then reference variable value?

Reply:
This could make sense on a new version :)
More customization is always a plus